frontends/election-manager/src/data/invalidElection.json

```diff
@@ -1 +1 @@
-This file is used to simulate errros purposefully not a json file.
+This file is used to simulate errors purposefully not a json file.
```

Comment:
It's possible the code here is trying to be cute, but...

Reply:
I don't think so. Better to fix it.